ci/blackbox/rdsbroker_test.go

```diff
@@ -436,17 +436,72 @@ var _ = Describe("RDS Broker Daemon", func() {
 				"postgres-micro-without-snapshot-10",
 			)
 		})
+	})
+
+	Describe("plan upgrade failure and recovery", func() {
```

Comment:
Is there any reason we can't easily squash this section in with the former one, recombining the `TestUpdatePlan` functions, making it simply omit the "recovery" if the final argument is `nil`?

Reason is I'd like to move away from thinking of "too much disk" as something that needs recovering from. If we're no longer considering having too much disk as being "off-plan" (as far as `compareDBDescriptionWithPlan()` is concerned), this is just a slightly dirtier case of what we were previously calling a "simple" failure - we're just making an extra check that the db isn't now stuck on its plan.